database-commons/src/main/java/io/cdap/plugin/db/sink/ETLDBOutputFormat.java

```diff
@@ -53,6 +53,12 @@
   // Batch size before submitting a batch to the SQL engine. If set to 0, no batches will be submitted until commit.
   public static final String COMMIT_BATCH_SIZE = "io.cdap.plugin.db.output.commit.batch.size";
   public static final int DEFAULT_COMMIT_BATCH_SIZE = 1000;
+  public static final String OPERATION_NAME = "io.cdap.plugin.db.operation.name";
+  public static final String RELATION_TABLE_KEY = "io.cdap.plugin.db.relation.table.key";
+
+  public static final String UPDATE = "update";
```

Comment:
create an enum for operation

Reply:
done

Reply:
getOperation should return enum value instead of string. e.g.   public Operation getOperation() {
    return Strings.isNullOrEmpty(operation) ? Operation.INSERT : Operation.valueOf(operation.toUpperCase());
  }

Reply:
done